src/synthesizer/emission_models/stellar/pacman_model.py

```diff
@@ -853,7 +856,7 @@ def _make_nebular(self, **kwargs):
             mask_op="<",
             fesc=self._fesc,
             fesc_ly_alpha=self._fesc_ly_alpha,
-            linecont=young_line_cont,
+            linecont=young_neb_line,
```

Comment:
Incorrect arg name, this is the origin of the error in the docs I think. `linecont` -> `nebular_line`

Reply:
Ah, does it do the strong concatenation automatically?

Reply:
Nah, it's that the keyword argument is checked for `nebular_line` so passing this to `linecont` will make the model try and make the same model twice. (I think anyway)